docs/source/agent-communication.md

```diff
@@ -69,33 +70,37 @@ name, doc = [
         # name of agent, used by adjudicator
         "agent": "aardvark",
         # unique ID of this ask / batch of suggestions
-        "publish_uid": "086ff562-c997-48c7-9fbb-9cbe3c362ff0",
+        "uid": "086ff562-c997-48c7-9fbb-9cbe3c362ff0",
```

Comment:
```suggestion
        "batch_uid": "086ff562-c997-48c7-9fbb-9cbe3c362ff0",
```

We have enough uids that it is worth prefxing them with which one they are.

Reply:
https://github.com/NSLS-II-PDF/mmm-experiments/pull/24/commits/3459e3c063ef6e5ee9e50fc19bd47e245ce06e1b